src/graphql/error.ts

```diff
@@ -153,7 +153,7 @@ export class DustAmountError extends CustomApolloError {
   constructor(errData: CustomApolloErrorData) {
     super({
       code: "ENTERED_DUST_AMOUNT",
-      message: `Use lightning to send amounts less than ${onChainWalletConfig.dustThreshold}`,
+      message: `Use lightning to send amounts less than ${onChainWalletConfig.dustThreshold} sats`,
```

Comment:
probably unrelated to this PR but surprising to see this message twice one time with `state` argument and another one with `onChainWalletConfig`

Reply:
Looks like one is the default message for that CustomApolloError from global config and the other is because we inject the global config (and `dustThreshold`) into the OnChainPaymentFlowBuilder